go.mod

```diff
@@ -22,10 +22,11 @@ require (
 	github.com/tendermint/tendermint v0.34.8
 	github.com/tendermint/tm-db v0.6.4
 	github.com/wealdtech/go-merkletree v1.0.0
-	golang.org/x/crypto v0.0.0-20210220033148-5ea612d1eb83 // indirect
+	golang.org/x/crypto v0.0.0-20210220033148-5ea612d1eb83
 	golang.org/x/term v0.0.0-20201210144234-2321bbc49cbf // indirect
 	google.golang.org/genproto v0.0.0-20210223151946-22b48be4551b
 	google.golang.org/grpc v1.36.0
+	google.golang.org/protobuf v1.25.0
```

Comment:
I'd bump the SDK dependency to the v0.42.LATEST and ensure the protobuf dependency matches what's used by the SDK's latest release (it should be v1.26.0)

Reply:
I'm going to open a pr dedicated to bumping version. there are some breaking changes in the 0.42.9 and I had trouble dealing with the intermediate versions on different branches.
Once we have the features on main (mainly this and the #153 swagger one) I'll got for the 0.43.0